### ISSUE TITLE
Error message is annotated with shortened filenames

### ISSUE THREAD

**Issue by ghost** [luvit/luvit#834]
The error message emitted by `assert()` is annotated by Luvit with the location where the `assert()` happened. The mentioned filenames are shortened however, making them almost useless.

Example:

```
$ cd /very/long/chain/of/directories
$ luvit tests/test-long-filename.lua 
Uncaught exception:
[string "/very/long/chain/of/directories/tests/test..."]:6: assertion failed!
stack traceback:
        [C]: in function 'assert'
        [string "/very/long/chain/of/directories/tests/test..."]:6: in function 'fn'
        [string "bundle:deps/require.lua"]:293: in function 'require'
        [string "bundle:main.lua"]:118: in function 'main'
        [string "bundle:init.lua"]:49: in function <[string "bundle:init.lua"]:47>
        [C]: in function 'xpcall'
        [string "bundle:init.lua"]:47: in function 'fn'
        [string "bundle:deps/require.lua"]:293: in function <[string "bundle:deps/require.lua"]:258>
```

Unless there is only one file named "test*" in the "tests" directory, this filename does not help at all.

**Comment by creationix** [comment]
This is the behavior of the default stack trace formatter in lua.  I think we could get around it by using xpcall and using our own custom stack trace formatter.

**Comment by urzds** [comment]
I also recommend using [Stack Trace Plus](https://github.com/ignacio/StackTracePlus). The additional information it provides proved to be quite useful in my previous projects.

**Comment by SinisterRectus** [comment]
Was fixed with 74e947ab8ee2af87c46021acb135bd2bdbceb626.
